modules/enabling-the-single-model-serving-platform.adoc

```diff
@@ -27,13 +27,14 @@ endif::[]
 . Enable pre-installed runtimes for the single-model serving platform as follows:
 .. In the left menu of the {productname-short} dashboard, click *Settings* -> *Serving runtimes*. 
 +
-The *Serving runtimes* page shows any custom runtimes that you have added, as well as the following pre-installed runtimes:
+The *Serving runtimes* page shows any custom runtimes that you have added and some pre-installed runtimes. 
```

Comment:
Just wondering if there's a better word to use that "some". 
Something like:
The *Serving runtimes* page shows any custom runtimes that you have added along with pre-installed runtimes.

Reply:
```suggestion
The *Serving runtimes* page shows preinstalled runtimes and any custom runtimes that you have added. 
```
@chtyler Does swapping these around help?

Reply:
looks good, em!